internal/tiltfile/starkit/fixture.go

```diff
@@ -12,23 +12,32 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.starlark.net/starlark"
+
+	"github.com/windmilleng/tilt/internal/testutils/tempdir"
 )
 
 // A fixture for test setup/teardown
 type Fixture struct {
-	tb         testing.TB
-	extensions []Extension
-	path       string
-	fs         map[string]string
-	out        *bytes.Buffer
-	useRealFS  bool // Use a real filesystem
+	tb               testing.TB
+	extensions       []Extension
+	path             string
+	temp             *tempdir.TempDirFixture
+	fs               map[string]string
+	out              *bytes.Buffer
+	useRealFS        bool // Use a real filesystem
+	loadInterceptors []LoadInterceptor
 }
 
 func NewFixture(tb testing.TB, extensions ...Extension) *Fixture {
+	temp := tempdir.NewTempDirFixture(tb)
+	temp.Chdir()
+
+	_ = os.Chdir(temp.Path())
```

Comment:
this is obsolete now? it's covered by the temp.Chdir

Reply:
Yup thanks